ionoscloud/data_source_dnsaas_record.go

```diff
@@ -0,0 +1,143 @@
+package ionoscloud
+
+import (
+	"context"
+	"fmt"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	dnsaas "github.com/ionos-cloud/sdk-go-dnsaas"
+	"log"
+	"strings"
+)
+
+func dataSourceDNSaaSRecord() *schema.Resource {
+	return &schema.Resource{
+		ReadContext: dataSourceRecordRead,
+		Schema: map[string]*schema.Schema{
+			"id": {
```

Comment:
maybe a UUID validation here and for zone_id

Reply:
@cristiGuranIonos done in 01e2b65f9fe5d96e6c21367e7fe3b2e7b6343ba2